.github/workflows/example-doc.yml

```diff
@@ -19,16 +19,40 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Build PDF
+      - name: Build draft PDF
         run: >-
           docker run \
               --volume "$(pwd):/data" \
               --user "$(id -u):$(id -g)" \
               openjournals/inara:latest \
-              -o contextpdf example/paper.md
+              -o "jats,contextpdf,crossref,preprint,pdf" example/paper.md
 
-      - name: Upload PDF
+      - name: Upload draft PDF
         uses: actions/upload-artifact@v3
         with:
-          name: paper
-          path: example/paper.context.pdf
+          name: draft
+          path: |
```

Comment:
Note to self: we should find a way to return the name of the generated files, so we don't have to maintain this manually.

Reply:
I tried to do it with a `**` glob pattern but that didn't work after a few tries so I went back to manually listing them